lib/src/util.dart

```diff
@@ -44,3 +45,22 @@ Future retry(Future f(), Duration interval, {Duration timeout}) async {
     }
   }, duration: timeout);
 }
+
+/// Returns an open port by creating a temporary Socket
+Future<int> getOpenPort() async {
+  ServerSocket socket;
+
+  try {
+    socket = await ServerSocket.bind(InternetAddress.LOOPBACK_IP_V4, 0);
+  } catch (_) {
+    // try again v/ V6 only. Slight possibility that V4 is disabled
+    socket = await ServerSocket.bind(InternetAddress.LOOPBACK_IP_V6, 0,
```

Comment:
I'm being paranoid. There are still systems that have IPv6 disabled. More likely a system has IPv4 and not v6 then the other way around.

I'm not 100% sure what happens on a system that is v4-only w/ the V6 API.